sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -369,6 +370,53 @@ def has_ontime_pane(self):
     return False
 
 
+class AfterSynchronizedProcessingTime(TriggerFn):
+  """A "runner's-discretion" trigger downstream of a GroupByKey
+  with AfterProcessingTime trigger.
+
+  AfterSynchronizedProcessingTime is experimental.
+  No backwards compatibility guarantees.
+  """
```

Comment:
Maybe make a note that this current implementation simply always fires.

Reply:
I mean, mostly triggers aren't Fns, so runners don't run this code at all. I've added a rather lengthy note explaining.

Reply:
Yeah, not any more. This is only used in the Python direct runners; this is fine.